src/nav/IconUnreadConversations.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import boundActions from '../boundActions';
+import { getUnreadPmsTotal, getUnreadHuddlesTotal } from '../selectors';
+import { IconPeople } from '../common/Icons';
+import { ComponentWithOverlay, UnreadCount } from '../common';
+
+const IconUnreadConversations = ({ unreadHuddlesTotal, unreadPmsTotal, color }) => {
+  const unreadCount = unreadHuddlesTotal + unreadPmsTotal;
+
+  return (
+    <ComponentWithOverlay
```

Comment:
@borisyankov have use ComponentWithOverlay here. Had to create a separate component to connect it to the redux store to fetch unread counts. Can this be avoided somehow?